### PR TITLE
Update README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by Dref360**
## Summary:

Rename BaaL to Baal.
Improve contribution guidelines as few people uses Poetry.
Added list of current maintainers.

## Checklist:

* [x] Your code is documented (To validate this, add your module to `tests/documentation_test.py`).
* [x] Your code is tested with unit tests.
* [x] You moved your Issue to the PR state.

### CODE REVIEW

**Comment by Dref360** [comment]
ping

**Comment by GeorgePearse** [comment]
> ping

Will review properly tonight! Cheers for the ping, had no idea this existed.

File: rfcs/01-semisupervised.md [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
 The goal of this feature is to remove the burden required to perform semi-supervised learning.
-Semi-supervised learning is widely popular in active learning and BaaL doesn't support it.
+Semi-supervised learning is widely popular in active learning and Baal doesn't support it.
 At first, we will **not** support label propagation, but it can be done in the future.
 
 ## Motivation
 
-Many papers propose to combine active learning and semi-supervised learning. Currently, it is not easy to perform SS learning with BaaL. One needs to know the internal representation of ActiveLearningDataset to do so.
+Many papers propose to combine active learning and semi-supervised learning. Currently, it is not easy to perform SS learning with Baal. One needs to know the internal representation of ActiveLearningDataset to do so.
 This RFC proposes a new API to make this easier.
```
Comment (GeorgePearse) [inline rfcs/01-semisupervised.md:22]: Will this just be label prop?

**Review by GeorgePearse — APPROVED** [review]

**Review by parmidaatg — APPROVED** [review]
looks good :)

File: rfcs/01-semisupervised.md (hunk 1 above)
Comment (Dref360) [reply rfcs/01-semisupervised.md:22]: I'm not familiar with any, and there is not much research around SSL in AL. We do have some light support for it, but we don't have a tutorial or anything.

**Pull request merged**